lib/solaar/listener.py

```diff
@@ -195,6 +195,9 @@ def _notifications_handler(self, n):
 		if not device_ready:
 			time.sleep(0.01)
 
+		if n.sub_id == 0x40 and not already_known:
+                        return # disconnecting something that is not known - nothing to do
```

Comment:
Please use the correct indentation, this is using spaces, the rest of the file is using tabs.

Reply:
Done.  I did modified the commit and force-pushed the result.